src/mod/applications/mod_av/avformat.c

```diff
@@ -382,7 +382,8 @@ static int mod_avformat_alloc_output_context2(AVFormatContext **avctx, AVOutputF
 #if (LIBAVCODEC_VERSION_INT < AV_VERSION_INT(58,7,100))
 		av_strlcpy(s->filename, filename, sizeof(s->filename));
 #else
-		s->url = strdup(filename);
+		s->url = av_mallocz(strlen(filename));	
```

Comment:
should we do strlen() + 1 ?

also need to put the strcpy after switch_assert

Reply:
ye, you are right,  i usually c++ string , it`s long time forget it , sorry for that.
i will recommit again.
thank you seven du